### PR TITLE
add search to also be cleared

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR adds setSearch to be passed through to the `reset filters` This will ensure the search is clear when someone clicks `clearAll`
#### Where should the reviewer start?
lists
#### What testing has been done on this PR?
local
#### How should this be manually tested?
check to see if the search is cleared out 
#### What are the relevant issues?
issue #1535 
#### Should this PR be mentioned in Design System updates?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: aries-site/src/examples/templates/filtering/FilteringCards.js [hunk 1]
```diff
@@ -254,6 +256,7 @@ const Filters = ({
     setName([]);
     setFilters(defaultFilters);
     setFiltering(false);
+    setSearch();
```
Comment (taysea) [inline aries-site/src/examples/templates/filtering/FilteringCards.js:259]: Since we default to `''` should this `setSearch('')` ?

Comment (halocline) [reply aries-site/src/examples/templates/filtering/FilteringCards.js:259]: Yes, good catch!

Comment (britt6612) [reply aries-site/src/examples/templates/filtering/FilteringCards.js:259]: Yes fixing now!

**Review by taysea — APPROVED** [review]
LGTM. Nice job.

**Pull request merged**